misc/docs-linter/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"golang.org/x/sync/errgroup"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+)
+
+var (
+	ErrEmptyPath = errors.New("you need to pass in a path to scan")
```

Comment:
Why is this exported?

Reply:
[8d1533a](https://github.com/gnolang/gno/pull/2125/commits/8d1533abec43f1bb16d2f0c9cb6eb75974657f86)